nessie/src/test/java/org/apache/iceberg/nessie/BaseTestIceberg.java

```diff
@@ -145,7 +147,8 @@ NessieCatalog initCatalog(String ref, String hash) {
             .put("ref", ref)
             .put(CatalogProperties.URI, uri)
             .put("auth-type", "NONE")
-            .put(CatalogProperties.WAREHOUSE_LOCATION, temp.toUri().toString());
+            .put(CatalogProperties.WAREHOUSE_LOCATION, temp.toUri().toString())
+            .put("client-api-version", apiVersion);
```

Comment:
should this be using the property from `NessieUtil`?

Reply:
Nope. The other configurations like `uri`, `ref` is not using properties from `NessieConfigConstant` because catalog properties are passed without prefix. 

`CONF_NESSIE_CLIENT_BUILDER_IMPL` is an exception which does not follow this style. We discussed this in Nessie public zulip channel and planning to deprecate it in future.